kcp/README.MD

```diff
@@ -1,22 +1,17 @@
 # Pipeline Services on managed KCP
 
 ## Clusters access
-The `login.sh -e staging` script will log you in into the various systems:
-1. KCP
-2. KCP workspace
-3. pipeline cluster
-4. ArgoCD
+Create a directory with the following files:
```

Comment:
Created a directory called `pipelines-service/kcp/work/kubeconfig` with the following files:

Reply:
I'm documenting how it currently works, the user could put the files under the work dir. However our login script create the appropriate dir, and I'm often nuking my work dirs, so it would be a pain to have to relogin every time.